### DISCUSSION TITLE
CodeHike - is it "limited" to React?

### DISCUSSION THREAD

**Discussion by Aiyush-G** [code-hike/codehike#90] [Q&A] [answered]
Hi, 

I'm not sure if this is a naive question, I was wondering if Code Hike can only be used with React, or could it be used with something like Svelte. I have come across this here: [example1](https://github.com/jpinho/svelte-react) , and [example2](https://github.com/jesseskinner/svelte-react), I also noticed on the README.md that: "First you need MDX v2 (or xdm). See MDX v2 docs on how to set it up." and I was wondering whether something like [this](https://mdsvex.pngwn.io) would work for Svelte instead?

I look forward to your response :)

**Comment by pomber** [comment, marked as answer]
It's a very good question. MDX already supports targets other than React, but Code Hike uses depends heavily on React. So **it is limited to React**.  

That said, you can import the whole MDX file as a React component. So if there are solutions to render React components from other frameworks this should work too. But I haven't tried.
